frontend/src/views/RouteFormView.vue

```diff
@@ -163,10 +166,15 @@ export default {
     },
   },
   created() {
-    this[actions.FETCH_CARS]();
+    this.fetchDrives();
```

Comment:
If we want to rely 100% on constants then you can change it to `this[actions.fetchCars]()`